linera-client/src/util.rs

```diff
@@ -32,3 +31,26 @@ pub async fn wait_for_next_round(stream: &mut NotificationStream, timeout: Round
     )
     .await;
 }
+
+macro_rules! impl_from_dynamic {
+    ($target:ty : $variant:ident, $source:ty) => {
```

Comment:
Does this work? Would probably look more natural.

```suggestion
    ($target:ty :: $variant:ident, $source:ty) => {
```

Reply:
It does not :( It's illegal to follow a `ty` with `::`, but `:` is okay.